libraries/common/cs/update-toolbox-xml.js

```diff
@@ -0,0 +1,14 @@
+export default function updateToolboxXML(tab) {
+  tab.waitForElement("[class*='gui_blocks-wrapper_']").then((blocksWrapper) => {
```

Comment:
We can add `!state.scratchGui.mode.isPlayerOnly` as a small optimization here

Reply:
Actually, this could be a simple `querySelector` - if the toolbox doesn't exist, there's no reason to update it.